extension/src/test/fixtures/expShow/kaggleDvcTitanic.ts

```diff
@@ -0,0 +1,1227 @@
+import { ExperimentStatus } from '../../../cli/dvc/contract'
```

Comment:
I think we will want to make a story out of this:

<img width="1728" alt="Screen Shot 2022-10-18 at 2 42 47 pm" src="https://user-images.githubusercontent.com/37993418/196330385-8301d2c0-ad85-4763-ab3b-06492e650bd5.png">

I will rearrange the fixtures.